src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -354,9 +355,24 @@ static int add_swap(const char *path) {
                 return 0;
         }
 
-        log_debug("Adding swap: %s", path);
+        if (streq_ptr(p->fstype, "crypto_LUKS")) {
+                r = unit_name_path_escape(p->node, &crypto_what);
+                if (r < 0)
+                        return r;
+
+                crypto_name = strjoin("swap-", crypto_what);
+                if (!crypto_name)
+                        return -ENOMEM;
```

Comment:
no, please revert to the old "swap" name, and document that in the spec, as I said.

the gpt-auto logic disables itself if things aren't obvious or unique anyway. hence use the simple, obvious name.

Reply:
No more than one swap partition is unsupported (as far as gpt-auto goes)?

I'll add a mention of that in the doc.

Reply:
yes, the gpt auto spec either cares only about the first (or about the "newest", as per version compare of the part label for root+usr fs), but never about more than one of a partition. Also, it's very defensively put together: if there's any configuration even remotely matching the object we are about to configure we step away.

Reply:
`swap` in particular is written a bit different:

> **All** swap partitions on the disk containing the root partition are automatically enabled.